spec/controllers/microsoft_controller_spec.rb

```diff
@@ -6,7 +6,7 @@
       with_modified_env AZURE_APP_ID: 'azure-application-client-id' do
         get '/.well-known/microsoft-identity-association.json'
         expect(response).to have_http_status(:success)
-        expect(response.body).to include '"applicationId":  "azure-application-client-id"'
+        expect(response.body).to include '"applicationId":"azure-application-client-id"'
```

Comment:
@tejaswinichile  if the header is a mandatory case, can we add a spec for the header as well .  other wise looks good to me.